### ISSUE TITLE
Remove unused translations

### ISSUE THREAD

**Issue by MichalBryxi** [simplabs/ember-intl-analyzer#122]
Currently the run of the plugin gives me a nice list:
```
❯ npx ember-intl-analyzer
[1/4] 🔍  Finding JS and HBS files...
[2/4] 🔍  Searching for translations keys in JS and HBS files...
[3/4] ⚙️   Checking for unused translations...
[4/4] ⚙️   Checking for missing translations...

 ⚠️   Found 211 unused translations!

   - Identification (used in cs.json)
   - Login (used in cs.json)
   - Password (used in cs.json)
   - Submit (used in cs.json)
   - admin.companies.address (used in cs.json)
...
```

But now what? I would have to go and manually remove every one of them. Simple shell script might do, but especially when using structured translations (like the last one in my example) this might get tricky.

**Comment by Turbo87** [comment]
interesting idea. I guess something like a `--fix` option might be nice.

**Comment by Mikek2252** [comment]
A fix option has now been released as of 3.1.0 so am closing this issue.
